### PR TITLE
Update hrv_OCRFixReplaceList.xml

### PR DESCRIPTION
**Pull request by diomed**
lots of fixes for Croatian language

### CODE REVIEW

File: Dictionaries/hrv_OCRFixReplaceList.xml [hunk 1]
```diff
@@ -1,256 +1,1115 @@
...
+<Word from="dali si" to="da li si" />
+<Word from="Dali si" to="Da li si" />
+<Word from="dedom" to="djedom" />
+<Word from="dejstvo" to="djelovanje" />
+<Word from="dela" to="djela" />
+<Word from="delo" to="djelo" />
+<Word from="deli" to="dijeli" />
+<Word from="delu" to="djelu" />
+<Word from="deluju" to="djeluju" />
+<Word from="djete" to="dijete" />               <Word from="Djete" to="Dijete" />
```
Comment (ivandrofly) [inline Dictionaries/hrv_OCRFixReplaceList.xml]: Bad format?

**Comment by diomed** [comment]
@ivandrofly I am sorry, what do you mean by that?

**Comment by ivandrofly** [comment]
> @ivandrofly I am sorry, what do you mean by that?

You should break line aster every  `/>`

**Comment by diomed** [comment]
why is this necessary? Program recognizes it nevertheless.

**Comment by xylographe** [comment]
It's not a problem. But, in the next "automated XML upkeep" it will be converted to standard format.
The effective change will be [exactly like this](https://github.com/xylographe/subtitleedit/commit/935f052ccd48c397f823eb4a534eed6b6d0b2db1).

**Comment by diomed** [comment]
wait, does this mean, those lines shall be deleted or placed under each other?

**Comment by xylographe** [comment]
No information shall be lost.

Lines containing more than one XML element will be split. Each element will start on a new line with indentation (leading spaces _before_ the opening tag). In general, this increases legibility, but doesn't effect the meaning of the XML elements, as all white space _between_ elements is ignored.

For example:
```
<Word from="menja" to="mijenja" />  <Word from="Menja" to="Mijenja" /> <!-- nije za regex! -->
```
becomes
```
    <Word from="menja" to="mijenja" />
    <Word from="Menja" to="Mijenja" />
    <!-- nije za regex! -->
```
or, if you prefer, it could become
```
    <!-- nije za regex! -->
    <Word from="menja" to="mijenja" />
    <Word from="Menja" to="Mijenja" />
```

**Comment by diomed** [comment]
how is this being executed, I mean what does this? 
or do you do that manually, or with some editor?

of course I'd prefer second option, which makes much more sense.

**Comment by xylographe** [comment]
[XmlDocument.Save Method](https://msdn.microsoft.com/en-us/library/system.xml.xmldocument.save%28v=vs.110%29.aspx)

**Comment by diomed** [comment]
@niksedk  why don't you merge this?

**Comment by niksedk** [comment]
sorry, I've been insanely busy here in May...
This looks like a massive change (very impressive!). How well have you tested? (I don't know Croatian so I cannot test). Be sure to test on many files that it does not change too much. It has happen before that too aggressive ocr replace lists has been submitted which does more harm than good!

**Comment by diomed** [comment]
I'm adding new things daily, been working on this for few weeks now.
I have tested it, and fixed as many errors I could find. I won't say it's 100% 
because I know I'm not unmistakable, yet it's better to have more things fixed than not.
I believe I didn't miss anything major.
You could merge, and we'll see - if any bugs appear I'll fix them in future versions.
I'll also tell community of Cro sub makers to test it for errors, if this gets merged & released in 3.4.7.

**Comment by niksedk** [comment]
Cool - I can see that a lot of work has gone into this :)

**Pull request merged**